recipes/kessler/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "kessler" %}
+{% set org = "kesslerlib" %}
+{% set version = "0.1.2.dev3" %}
```

Comment:
This one falls under https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Reply:
@ocefpaf  Thanks for looking into this. Is there some action I need to take for this?

Reply:
Either ship a previous stable version, or wait for a future stable,  or follow the instructions in that doc above to publish this one in a dev sub-channel. Choose what you think is more appropriate for your goals.